libkbfs/block_retrieval_queue.go

```diff
@@ -584,36 +428,21 @@ func (brq *blockRetrievalQueue) FinalizeRequest(
 		// Since we created this channel with a buffer size of 1, this won't
 		// block.
 		req.doneCh <- err
-		doneChans = append(doneChans, r.deepPrefetchDoneCh)
-		cancelChans = append(cancelChans, r.deepPrefetchCancelCh)
 	}
 	// Clearing references to the requested blocks seems to plug a
 	// leak, but not sure why yet.
+	// TODO: strib fixed this earlier. Should be safe to remove here, but
```

Comment:
FYI, I checked out this branch and deleted this line, and was able to observe the memory leak again.  I guess it was unrelated to the goroutine that got removed.  Still mystified.

Reply:
![mind blown](https://media2.giphy.com/media/26ufdipQqU2lhNA4g/giphy.gif)